### ISSUE TITLE
"Enter page number" input field not accessible by keyboard

### ISSUE THREAD

**Issue by plannero** [edrlab/readium-desktop#3325]
Hi!

I cannot move keyboard focus to the input field "Enter page number" using keyboard only.
This field probably needs a tab index.

Using Thorium version 3.3.0 on MacOS. 

<img width="553" height="240" alt="screen dump" src="https://github.com/user-attachments/assets/c360948e-2600-49bf-b884-a8df9b4dcd0d" />

**Comment by gautierchomel** [comment]
This may happen because there is no page in this ebook. See [Deactivate Go To Page navigation when it is not available](https://github.com/edrlab/thorium-reader/issues/3284)
[accessibility](https://github.com/edrlab/thorium-reader/issues?q=is%3Aissue%20state%3Aopen%20page%20label%3Aaccessibility)

**Comment by danielweck** [comment]
Mouse cursor "not available" and colour fade?
CC @arthur-lemeur

**Comment by gautierchomel** [comment]
Same as #3284
